spec/io_spec.rb

```diff
@@ -146,4 +146,28 @@
     expect(o).to eq(m)
     expect(o).not_to eq(n)
   end
+
+  it "reads csv file containing a single dense integer matrix" do
+    n = NMatrix::IO::Csv.load 'spec/2x2_dense_int.csv', dtype: :float64
+    m = NMatrix.new(2, [1, 2, 5, 8], dtype: :int64)
+    expect(n).to eq(m)
+  end
+
+  it "reads csv file containing a single dense float matrix" do
+    n = NMatrix::IO::Csv.load 'spec/2x2_dense_float.csv'
+    m = NMatrix.new(2, [1.2, 2.3, 5.1, 12.2], dtype: :float64)
+    expect(n).to eq(m)
+  end
```

Comment:
How can you be sure that these tests work by using `libcsv` and are not falling back to Ruby `CSV`?